homeassistant/components/nasweb/strings.json

```
@@ -0,0 +1,46 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add NASweb device",
+        "description": "{nasweb_schema_img}NASweb combines the functions of a control panel and the ability to manage building automation. The device monitors the flow of information from sensors and programmable switches and stores settings, definitions and configured actions.",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "ha_address": "Home Assistant Address"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "missing_status": "Did not receive status from NASweb, check logs",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "issues": {
+    "invalid_authentication": {
+      "title": "[%key:common::config_flow::error::invalid_auth%]",
+      "description": "{domain_name}[{device_name}]: Invalid username/password. Most likely user changed password or was removed. Delete this entry and create new one with correct username/password."
+    },
+    "internal_error": {
+      "title": "Internal Error",
+      "description": "{domain_name}[{device_name}]: Something isn't right with device internal configuration. Try restarting device and HomeAssistant. If Issue persist contact support: support@chomtech.eu"
```

Pass in the email address via translation_placeholders:
```suggestion
      "description": "{domain_name}[{device_name}]: Something isn't right with device's internal configuration. Try restarting the device and HomeAssistant. If the issue persists contact support at {support_email}"
```